app/models/project.rb

```diff
@@ -212,8 +212,12 @@ def asset_count(session_id:)
     values.fetch(:total_file_count, 0)
   end
 
+  def self.default_storage_unit
+    "KB"
+  end
+
   def self.default_storage_usage
-    "0 KB"
+    "0 {default_storage_unit}"
```

Comment:
I believe this needs to change
```suggestion
    "0 #{default_storage_unit}"
```